www/docs/src/content/docs/index.mdx

```diff
@@ -5,8 +5,9 @@ template: splash
 hero:
   tagline: A dedicated CMS for Astro Studio. Built from the ground up by the Astro community.
   image:
-    file: ../../../../assets/studioCMS.png
-    alt: Astro StudioCMS
+    alt: Astro StudioCMS Logo
+    dark: ../../../../assets/studioCMS.png
```

Comment:
Are path aliases allowed here?

`~/assets/studioCMS.png`

Reply:
I dont think that will work since the assets folder its using is outside of the actual docs folder.... its looking at `www/assets`, not `www/docs/src/assets`....